dev/php.ini

```diff
@@ -1,17 +1,28 @@
 [PHP]
-; Prod settings -- see ../prod/php.ini
+; Prod settings
 date.timezone = UTC
-zend.assertions	= 0
-upload_max_filesize = 24M
-post_max_size = 32M
 file_uploads = On
-
-; CI settings - see ../ci/php.ini
-; TODO Change memory_limit to prod value.
-memory_limit = 1024M
-max_execution_time = 0
-zend.assertions = 1
+display_errors = Off
+display_startup_errors = Off
+max_execution_time = 120
+max_input_time = 900
+max_input_vars = 10000
+opcache.enable = 1
+opcache.enable_cli = 1
+opcache.interned_strings_buffer = 16
+opcache.max_accelerated_files = 65407
+opcache.max_file_size = 0
+opcache.memory_consumption = 128
+opcache.revalidate_freq = 0
+opcache.validate_timestamps = Off
```

Comment:
I believe this hinders development because code changes won't be reflected until an op-cache reset which you don't want to do after every file change.

Reply:
The initial idea was given by Jochem. Also, we want to have the same behaviour as production on our local, I do agree with your point, but would it be possible for your to suggest a change where we can achieve both?